src/renderer/store/playback/reducer.ts

```diff
@@ -60,10 +66,27 @@ const playbackReducer: Reducer<ApplicationStore['playback'], Action<any>> = (
           timeState.maxDuration
         ),
         lastUpdated: timeState.lastUpdated,
+        rangeOverride: playback.rangeOverride,
       };
     }
   }
 
+  if (action.type === SET_RANGE_OVERRIDE) {
+    const { rangeOverride } = action.payload as SetRangeOverridePayload;
+
+    return {
+      ...playback,
+      rangeOverride,
+    };
+  }
+
+  if (action.type === CLEAR_RANGE_OVERRIDE) {
+    return {
+      ...playback,
```

Comment:
lol you did it here

Reply:
yeah that's because this is a new action, he was just following what the previous author did for the other actions

Reply:
Chloe has my back :muscle: beat that @Mhudlark